### DISCUSSION TITLE
Is it possible to use multiple glyph for the single field?

### DISCUSSION THREAD

**Discussion by dhruvinsh** [caksoylar/keymap-drawer#48] [Q&A] [answered]
for example I want to convey `LS(TAB)` to have multiple glyph showing Ctrl+Tab, but seems like parser can not work with multiple glyph, is it implemented in any way for now? 

Edit:
looks like regex can only work with single glyph `_glyph_name_re = re.compile(r"\$\$(?P<glyph>.*)\$\$")` is this correct?

**Comment by caksoylar** [comment, marked as answer]
It is currently not supported and the glyph spec has to be the only text in the field, yes. This is because the horizontal placement of multiple glyphs (or glyph+text) would need to be handled specially. My guess is it will be a bit painful, where we would need to use the width of the glyph and manually place the elements, centering them in the key and placing side by side with a small gap. 

That being said, this was asked a couple times before so it is a feature request that I can spend some time looking into.
